### ISSUE TITLE
selected event not triggered on mouse click 

### ISSUE THREAD

**Issue by asimhashmi** [oferh/ng2-completer#302]
hey, guys, ng2-completer is not emitting the selected event on mouse click, it only emits when I use the arrow keys on the suggestion list and press enter.
 `<ng2-completer class="col-10"  (selected)="getNewChat($event)" [datasource]="completerData"></ng2-completer>`
and
`getNewChat(selected: CompleterItem) {
    console.log('item selected');
  }`
it only triggers this method when I select the item with arrow keys and press Enter. On mouse click the method is not triggered
here is the result of ng -v 
`@angular/cli: 1.2.4
node: 8.1.2
os: win32 x64
@angular/animations: 4.3.1
@angular/common: 4.3.1
@angular/compiler: 4.3.1
@angular/core: 4.3.1
@angular/forms: 4.3.1
@angular/http: 4.3.1
@angular/platform-browser: 4.3.1
@angular/platform-browser-dynamic: 4.3.1
@angular/router: 4.3.1
@angular/upgrade: 4.3.1
@angular/cli: 1.2.4
@angular/compiler-cli: 4.3.1
@angular/language-service: 4.3.1
`

**Comment by oferh** [comment]
can you provide a plunker? it's working well in the demo and [here](https://plnkr.co/edit/sVnfpBiEb5jBdtul4ls9?p=preview)

**Comment by asimhashmi** [comment]
I played it a little, it was responding on double click, not single, then I restarted everything, I think my browser or some libraries which I imported were messing with it, now working fine

**Comment by asimhashmi** [comment]
I tried to reproduce it in plunker, but in the plunker, it's working fine but in my project, it sometime get selected item on single click and sometime it doesn't work. any idea what's messing with it, I'm also using
angular-bootstrap-md, font-awesome and some other modules

**Comment by malchijah** [comment]
@asimhashmi Where you able to fix this?

**Comment by malchijah** [comment]
@asimhashmi Were you able to fix this?

**Comment by asimhashmi** [comment]
nope, but as a workaround, if you double click on the item to select, it always works

**Comment by rishabh010189** [comment]
Did anyone found a fix for the same?
I am also experiencing the same issue, it is working fine in many places but fails to work as expected in a few cases, although the code and usage are exactly the same.

**Comment by rishabh010189** [comment]
seems to be an unusual bug. The completer is working fine on some places in the same app/ component, but not working on others, despite the same approach and code.

**Comment by romanpon** [comment]
I have the same issue and found out that selected event is not working when I am clicking on precision touchpad with windows gesture where you can just tap on touchpad and it will send left click but it is working when I am using touchpad button or when I am using mouse

Could anyone verify the same?

**Comment by dnsh4989** [comment]
I have the same issue and it's working fine when double clicked.

**Comment by MrNocTV** [comment]
It's 2020 and I still have this issue.

**Comment by MrNocTV** [comment]
I've just got the answer on Stackoverflow: https://stackoverflow.com/questions/60071145/ng2-completer-selected-event-not-triggered-on-mouse-click/60071654#60071654
